### ISSUE TITLE
Vagrant box metadata JSON wrong Content-Type

### ISSUE THREAD

**Issue by schroedingersZombie** [flynn/flynn#4051]
When trying to run `vagrant up` I get the following error:
```You specified a box version constraint with a direct box file
path. Box version constraints only work with boxes from Vagrant
Cloud or a custom box host. Please remove the version constraint
and try again.
```
I suspect that is caused because `https://dl.flynn.io/vagrant/flynn-base.json` has it's content-typer set to `text/plain` if I host the file myself with content-type `application/json` this step works like expected.
I also tested with self served `text/plain` - same error as with the official server.

**Comment by petersondrew** [comment]
I worked around this temporarily by downloading the json file and serving it locally using node-static (`npm install -g node-static`), updating my vagrant file to look at localhost, and serving the file locally using `static -H '{"Content-Type": "application/json"}'`.

**Comment by arthurlogilab** [comment]
even with this workaround I get the following error : 

```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'flynn-base' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: > 0
16:2:21 [200]: /flynn-base.json
==> default: Loading metadata for box 'http://localhost:8080/flynn-base.json'
    default: URL: http://localhost:8080/flynn-base.json
16:2:21 [200]: /flynn-base.json
==> default: Adding box 'flynn-base' (v20170417.2) for provider: virtualbox
    default: Downloading: https://dl.flynn.io/vagrant/boxes/flynn_v20170417.2_ubuntu-xenial_virtualbox.box
==> default: Box download is resuming from prior download progress
    default: Progress: 39% (Rate: 287k/s, Estimated time remaining: 0:26:16)^R
    default: Calculating and comparing box checksum...
The checksum of the downloaded box did not match the expected
value. Please verify that you have the proper URL setup and that
you're downloading the proper file.

Expected: 5deb51ae17c7862de9cc4ba7fe5593d2aa08067e2bdc457999deb66eaffbe791
Received: be409f5d7b93853afe1e9a0dc4201ac29c5a6c84290248ee6c488e57f3527181

```

**Comment by petersondrew** [comment]
Yeah, I opened a separate issue for that #4055

**Comment by titanous** [comment]
I fixed this manually and filed #4061 to make sure it doesn't happen again.
